src/main/java/step2/domain/LottoStatistics.java

```diff
@@ -0,0 +1,24 @@
+package step2.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+public class LottoStatistics {
+
+    private LottoStatistics() {
+
+    }
+
+    public static Map<Integer, Integer> calculateWinningCountStatistics(LottoResults lottoResults) {
+        Map<Integer, Integer> result = new HashMap<>();
+        IntStream.range(LottoConfig.WINNING_COUNT_FROM, LottoConfig.WINNING_COUNT_TO + 1)
```

Comment:
알고계시겠지만,
```suggestion
        IntStream.rangeClosed(LottoConfig.WINNING_COUNT_FROM, LottoConfig.WINNING_COUNT_TO)
```
으로 사용할 수도 있습니다:)